packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -0,0 +1,230 @@
+import styled from 'styled-components';
+
+import { useSelector } from 'src/hooks/suite/useSelector';
+import { AddressDisplayOptions, selectAddressDisplay } from 'src/reducers/suite/suiteReducer';
+import {
+    selectDeviceInternalModel,
+    selectDeviceUnavailableCapabilities,
+} from '@suite-common/wallet-core';
+import { DeviceModelInternal } from '@trezor/connect';
+import { Icon, variables } from '@trezor/components';
+import { Translation } from './Translation';
+
+const Content = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Display = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 16px;
+    min-height: 134px;
+    width: 100%;
+    background: #000000;
+    padding: 25px 12px;
+`;
+
+const Text = styled.div<{ isPixelType: boolean; areChunksUsed?: boolean }>`
+    font-family: ${props => (props.isPixelType ? 'PixelOperatorMono8' : 'RobotoMono')};
```

Comment:
We usially deconstruct props everywhere 🙏  

```suggestion
    font-family: ${({ isPixelType }) => (isPixelType ? 'PixelOperatorMono8' : 'RobotoMono')};
```